doc/arkode/guide/source/Usage/User_callable.rst

```diff
@@ -1094,6 +1099,30 @@ Set max number of constraint failures             :c:func:`ARKodeSetMaxNumConstr
    .. versionadded:: 6.1.0
 
 
+.. c:function:: int ARKodeSetStepDirection(void* arkode_mem, sunrealtype stepdir)
```

Comment:
I'm surprised that this uses `sunrealtype` instead of `int`.

Reply:
Usually the direction comes from something like `tf - t0` which is a `sunrealtype`. A cast to int seems unnecessary and a source of sneaky bugs from converting double to int (undefined behavior if greater than INT_MAX)

Reply:
Note to self: might be able to use this function in the adjoint stepping.

Reply:
`signbit` splits by negative and non-negative but doesn't indicate when 0. For that reason, I don't think we could use it directly.